electron-builder-nightly.yml

```diff
@@ -33,11 +33,13 @@ nsis:
   oneClick: false
   perMachine: true
   allowToChangeInstallationDirectory: true
+  installerIcon: build/windows/installer.ico
+  installerSidebar: build/windows/installerSidebar.bmp
+  uninstallerIcon: build/windows/uninstaller.ico
+  uninstallerSidebar: build/windows/uninstallerSidebar.bmp
 
 files:
   # Include files
   - .webpack/**/*
-  - node_modules/**/*
-  - package.json
```

Comment:
While unrelated to the Windows CI build fix, these two lines are not in the release config, so I removed them too.
It shouldn't change anything as `electron-builder` includes both by default so they don't need to be listed AFAIK.